examples/examples.py

```diff
@@ -33,57 +52,146 @@
 
 """You can always check the user entity that is logged in
 by entering the user ID and token"""
-mf_resp = mfsdk.users.get(id="<user_id>", token="<user_token>")
+mf_resp = mfsdk.users.get(user_id="<user_id>", token="<token>")
 if mf_resp.error.status == 0:
     print(mf_resp.value)
 else:
     print(mf_resp.error.message)
 
-"""Updating user entities in the database"""
-mf_resp = mfsdk.users.update(
-    user_token="<user_token>", user={"metadata": {"foo": "bar"}}
-)
+"""Updates user entities in the database"""
+user = {
+    "id": "<user_id>",
+    "name": "<user_name>",
+    "metadata": {
+        "foo": "bar"
+    }
+}
+mf_resp = mfsdk.users.update(user_token="<token>", user=user)
 if mf_resp.error.status == 0:
     print(mf_resp.value)
 else:
     print(mf_resp.error.message)
 
-"""You can get all users in the database by calling the get_all () function"""
-mf_resp = mfsdk.users.get_all(
-    query_params={"offset": 1, "limit": 5}, admin_token="<admin_token>"
-)
+"""Updates user identity in the database"""
+user = {
+  "credentials": {
+    "identity": "<user_identity>",
```

Comment:
should be a new identity, `<user_identity>` suggests the user enter the current identity